code/game/objects/items/kitchen.dm

```diff
@@ -102,6 +102,45 @@
 	. = ..()
 	AddElement(/datum/element/easily_fragmented, PLASTIC_BREAK_PROBABILITY)
 
+/obj/item/knife/kitchen/silicon
+	name= "Kitchen Toolset"
+	icon_state = "SiliKnife"	
+	desc = "A breakthrough in synthetic engineering, this tool is a knife programmed to dull when not used for cooking purposes, and can exchange the blade for a rolling pin"
+	force = 0
+	throwforce = 0
+	sharpness = SHARP_EDGED
+	usesound = 'sound/items/drill_use.ogg'
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb_continuous = list("prods", "whiffs", "scratches", "pokes")
+	attack_verb_simple = list("prod", "whiff", "scratch", "poke")
+	tool_behaviour = TOOL_KNIFE
+
+/obj/item/knife/kitchen/silicon/examine()
+	. = ..()
+	. += " It's fitted with a [tool_behaviour == TOOL_KNIFE ? "knife" : "rolling pin"] head."
```

Comment:
Okay, I know why you did it like this, but it should really be like this because of how `tool_behaviour` flags compile out to strings.

```suggestion
	. += " It's fitted with a [tool_behaviour] head."
```

Then, you should go to the file where we have `#define TOOL_ROLLINGPIN "rollingpin"` and swap it to `#define TOOL_ROLLINGPIN "rolling pin"`. This shouldn't have any adverse effects on the code pretty much anywhere (by design), and the benefit is clear.